src/dipole.js

```diff
@@ -224,12 +212,8 @@ class Computed {
     }
 
     _unsubscribe(subscriber) {
-        // do not react to unsubscribes when in NOTIFYING state,
-        // as _subscribers HashSet is broken
-        if (this._state === states.NOTIFYING) return;
```

Comment:
It's better to leave the check here because it makes behavior during notification stage more predictable

Reply:
I don't think I removed this intentionally, I will bring it back 🙂